showcase/app/templates/components/advanced-table.hbs

```diff
@@ -930,6 +930,32 @@
 
   <Shw::Divider @level={{2}} />
 
+  <Shw::Text::H3>Wide container with a narrow AdvancedTable</Shw::Text::H3>
+
+  <div class="shw-component-advanced-table-narrow-wrapper">
+    <Hds::AdvancedTable
+      @isSelectable={{true}}
+      @onSelectionChange={{this.onMultiSelectUsersSelectionChange__demo3}}
+      @model={{this.model.userData}}
```

Comment:
[suggestion] since this table is used only to demo the case of a narrow table with a select column, you could use a simpler data model here (see for example the "Large AdvancedTable with overflowing container") so that we don't show 39 rows, in a page which is already extremely long